packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -335,4 +335,27 @@ export const Query = {
       sellers,
     }
   },
+  profile: async (_: unknown, { userId }: QueryProfileArgs, ctx: Context) => {
```

Comment:
@hellofanny . There is a difference between `profileId` and `userId`. `userId` is related to the entity of a vtex-id. `profileId` is related to the entity of profile system. If you decode the vtexIdClientAuthCookie you will see something like `"userId": "f3e731fd-6e0d-4150-b570-fb56b336340e"`. But you are using the `session.person.id`. Could you use the `profileId` name here ?

Reply:
as we discussed, I'll just rename to `id` for now. Then we can decide how to proceed.